lib/jnpr/junos/utils/fs.py

```diff
@@ -268,30 +270,34 @@ def _decode(fs):
     # directory_usage - filesystem directory usage
     # -------------------------------------------------------------------------
 
-    def directory_usage(self, path="."):
+    def directory_usage(self, path=".", depth=0):
         """
         Returns the directory usage, similar to the unix "du" command.
 
-        :returns: approximate directory usage, including subdirectories, in bytes
+        :returns: dict of directory usage, including subdirectories if depth > 0
         """
-        rsp = self._dev.rpc.get_directory_usage_information(path=path, depth="0")
+        BLOCK_SIZE = 512
 
-        used_space = rsp.findtext("directory/used-space")
+        rsp = self._dev.rpc.get_directory_usage_information(path=path, depth=str(depth))
```

Comment:
Add `normalize=True` argument. So, line becomes:
`rsp = self._dev.rpc.get_directory_usage_information(path=path, depth=str(depth), normalize=True)`
This avoids having to call `strip()` on lines 291-293.